coordinator/coordinator.go

```diff
@@ -39,6 +39,23 @@ func templateReplace(template string, replacements map[string]string) (string, e
 	return formattedString, nil
 }
 
+func getSourceMapping(template string, replacements map[string]string) ([]provider.SourceMapping, error) {
```

Comment:
So this is to reconcile the fact that the template replace will apply sanitized table names to the query but we still need to store the unsanitized names?

Reply:
Basically, this will be a temporary fixed that we can leverage for Spark SQL & Dataframes. Once we have time to change all the providers, this will be modified to create the list of `{"{{ template }}" : "table" }` and we can use this instead of the template replace.